BEIMA.Client/src/services/AllDevicesReport.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import * as Constants from '../Constants.js';
+const API_URL = process.env.REACT_APP_API_URL;
+
+/**
+ * GET API call to get a report of all devices in the database
+ *
+ * @return the report in the form of bytes
+ */
+const GetAllDeviceDevicesReport = async() => {
+  const deviceReportCall = await axios.get(API_URL + "report/devices", {responseType: 'blob'}).catch(function (error) {
```

Comment:
We should add the auth token to this call, as I have a PR up right now that is going to require it.

Reply:
I will make a task for that when the full_authentication_service branch gets merged into main.

Reply:
Task is #486